editor/components/inserter/menu.js

```diff
@@ -280,15 +280,21 @@ export class InserterMenu extends Component {
 		return this.renderCategories( visibleBlocks );
 	}
 
-	interceptArrows( event ) {
+	onTabbableEvent( event ) {
+		// If a tab (Recent, Blocks, …) is focused, pressing the down arrow
+		// moves focus to the selected panel below.
```

Comment:
What would the default behaviour be?

Reply:
The default is to let the events propagate. In this instance, the down-arrow keypress would reach the editor and move the cursor out of place and stealing focus from the inserter. This was originally prevented with the `interceptArrows` event handler (https://github.com/WordPress/gutenberg/pull/3281/commits/ac75a8aa13ea4f5108e7aee9ebff0881fcb97194). Some of that logic is now being adapted into `onTabbableEvent`.